server/WAPI.rb

```diff
@@ -66,7 +66,7 @@ def initialize(application)
     @uniqname = application['uniqname']
 
     @renewal = WAPI.build_renewal(@key, @secret)
-    logger.info("WAPI: #{__LINE__}: initialize WAPI with #{@api_prefix}")
+    logger.info("#{self.class.to_s}:#{__method__}:#{__LINE__}: initialize WAPI with #{@api_prefix}")
```

Comment:
I wonder whether it is possible to make a util routine for log String format: "#{self.class.to_s}:#{__method__}:#{__LINE__}"